### PR TITLE
Prime: add furnace bsj and spider standable documentation for PAL

### PR DESCRIPTION
**Pull request by JeffGainsNGames**
- also adds a video for Control tower OoB

### CODE REVIEW

File: randovania/games/prime1/logic_database/Phendrana Drifts.txt [hunk 1]
```diff
@@ -1810,6 +1810,7 @@ Extra - aabb: [-66.96835, -841.40247, 152.18471, 56.02846, -730.42065, 221.62587
               Space Jump Boots and Slope Jump (Intermediate)
               Any of the following:
                   Single Room Out of Bounds (Advanced)
+                  # https://youtu.be/NTcnaEYU9Ts?si=J3VhbqtPjK6KNzUj
```
Comment (Miepee) [inline randovania/games/prime1/logic_database/Phendrana Drifts.txt:1813]: https://youtu.be/NTcnaEYU9Ts

lets remove the tracking parameter here.

File: CHANGELOG.md [hunk 2]
```diff
@@ -55,6 +55,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 #### Logic Database
 
+- Added: Videos added to the logic database.
```
Comment (Miepee) [inline CHANGELOG.md:58]:
```suggestion
- Added: 3 Videos added to the logic database.
```

Comment (JeffGainsNGames) [reply CHANGELOG.md:58]: Is it worth evening adding a log for this? I know it was done it the past but does this really add anything? (also why I excluded the count)

Comment (Miepee) [reply CHANGELOG.md]: in this case i would probably just do it to have consistency with the past.

Count is nice to have to a) not have it be misinterpreted as "there were no videos before, now there are" (although you could also just rephrase it to get around it) and b) give an approximation on how useful it is. E.g. seeing `240 videos added` will probably give you more incentive to look into the database for videos again than `2 videos added`

Comment (Miepee) [reply CHANGELOG.md]: Since our videos also show up on randovania.org, it also helps there too ig.
